test/acsengine.deploy

```diff
@@ -101,6 +105,12 @@ uksouth ukwest"
             // Generate reports
             try {
               junit("${junit_dir}/**/junit.xml")
+              if(sendReport && (sendTo != "")) {
```

Comment:
I think there's a way to query the current build status - as in "failed" or "succeeding". You might consider that instead of the `sendReport` sentinel.